### DISCUSSION TITLE
How to uninstall a nightly build package (OSX)?

### DISCUSSION THREAD

**Discussion by ubaldot** [OpenModelica/OpenModelica#11032] [Q&A]
I have installed the latest nightly build taken from [here](https://build.openmodelica.org/mac/pkg/arm64/nightly/) (2023-05-02 03:06).

I wish to completely remove it but in the installation package there is no option for uninstalling it. Would that be enough to trash the  `/opt/omc` folder? Or shall I also remove other files around? If so, which ones? 

Many thanks!

**Comment by mahge** [comment]
Removing the `opt/omc/` folder should remove everything installed by the `productbuild` package.

You may still have temporary files that are created in some specific user folders (e.g. by OMEdit simulation runs). Any Modelica packages you installed using the OpenModelica package manager will also remain in your user directory.

**Reply by ubaldot** [reply]
Ok, then I will trash `opt/omc`. Where I can find exactly such temporary files in my user folder? 

Thanks again! :)

**Reply by mahge** [reply]
@adeas31 @hkiel would you happen to know where the temporary files and installed Modelica libraries will end up on macOS?

**Reply by hkiel** [reply]
OMNotebook tells me with `cd()`
```
"/private/var/folders/2c/_y0p00j51392xxcwc546l6cc0000gn/T/OpenModelica"
```
(a temporary folder)

I don't know where OMEdit stores its files.
